html/gizmodo.html

```diff
@@ -5,9 +5,10 @@
     <title>GIZMODO.com</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="../stylesheets/gizmodo.css">
+    <link rel="stylesheet" type="text/css" href="../css/gizmodo.css">
     <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
 </head>
 
 <body>
```

Comment:
1. Use h tags in a hierarchical manner
2. <a href="">Eve Peyser</a> - is it a link?
3. <div>
                            <img src="../images/comment.png" alt="comments-count">
                            <span class="comment-count">23</span>
                        </div> - can we use span instead of div
4. <p class="odd-description">
                        <a href="">Sometimes, all a man has in this world is his name, but according to Intel, antivirus pioneer and living cautionary tale John McAfee <em>doesnt even have that</em></a>
                    </p> - is it a link?
5. <section class="metadata"> - can use meta-data